evm-contracts/src/v0.6/dev/PrepaidAggregator.sol

```diff
@@ -535,7 +565,7 @@ contract PrepaidAggregator is AggregatorInterface, Owned {
    * Private
```

Comment:
While this comment isn't part of the PR, I think we should remove it.

Reply:
That's part of a bigger pattern that I think we should talk about separate from this PR.

Reply:
OK.